source/includes/steps-create-manually-windows-service-for-mongodb.yaml

```diff
@@ -53,10 +53,10 @@ action:
   - pre: "Create the MongoDB service."
     language: powershell
     code: |
-      sc.exe create MongoDB binPath= "\"C:\Program Files\MongoDB 2.6 Standard\bin\mongod.exe\" --service --config=\"C:\Program Files\MongoDB 2.6 Standard\mongod.cfg\"" DisplayName= "MongoDB 2.6 Standard" start= "auto"
+      sc.exe create MongoDB binPath= "\"C:\mongodb\mongod.exe\" --service --config= \"C:\mongodb\mongod.cfg\"" DisplayName= "MongoDB" start= "auto"
     post: |
       ``sc.exe`` requires a space between "=" and the configuration values
-      (eg "binPath= "), and a "\" to escape double quotes.
+      (eg "binPath= "), and a "\\" to escape double quotes.
```

Comment:
but then shouldn't you have double \\ in the code to escape the double quotes?  -- I'm going to merge this since it's already manually merged into 2.6.  Please fix as needed in both master and v2.6